registration/include/pcl/registration/transformation_estimation_lm.h

```diff
@@ -88,7 +88,7 @@ class TransformationEstimationLM
   , tmp_tgt_(src.tmp_tgt_)
   , tmp_idx_src_(src.tmp_idx_src_)
   , tmp_idx_tgt_(src.tmp_idx_tgt_)
-  , warp_point_(src.warp_point_){};
+  , warp_point_(src.warp_point_) {};
```

Comment:
```suggestion
  , warp_point_(src.warp_point_){};
```

Reply:
I am confused, i ran
```
 clang-format -style=file registration/include/pcl/registration/transformation_estimation_lm.h -i
``` 
which does not suggest any changes.

Reply:
Then you are probably not using clang-format 14. That is what the formatting check uses. Unfortunately, different versions of clang-format sometimes format things differently.

Reply:
my version is 19, perhaps too new.